lib/BackgroundJob/CheckOpenTickets.php

```diff
@@ -43,8 +43,8 @@ class CheckOpenTickets extends TimedJob {
 	protected $logger;
 
 	public function __construct(ITimeFactory $time,
-								JiraAPIService $jiraAPIService,
-								LoggerInterface $logger) {
+		JiraAPIService $jiraAPIService,
+		LoggerInterface $logger) {
```

Comment:
Since you know Php >= 8.0 will be used you could use attribute declaration in construct params. Not very important though.